src/meta-srv/src/handler/failure_handler/runner.rs

```diff
@@ -156,6 +156,12 @@ impl FailureDetectRunner {
                         })
                         .collect::<Vec<RegionIdent>>();
 
+                    if region_failover_manager.is_maintenance_node().await.unwrap() {
```

Comment:
I think we should check the maintenance mode in `RegionFailoverManager::do_region_failover`. Failure detector can keep working during maintenance mode, just the failover not to be triggered. It's easier.

Reply:
ok, i used to think that failure detector just do region failover, so i move the checker to here